.github/workflows/ci.yml

```diff
@@ -4,7 +4,6 @@ on:
   pull_request:
   push:
     branches: [ main ]
-  workflow_call:
```

Comment:
we needed this when we were calling this workflow from `deploy-to-staging.yml` but we’re not anymore

Reply:
is this getting called on push to `main` branch as well? if so, are we running the tests twice on main?

Reply:
🤔 maybe it’s not important to run the linter everytime we push

Reply:
it can be a separate PR though

Reply:
🤔 i am leaning separate PR. we originally had this called as a workflow but because we only want the slack notification for non-pr environments, we’re not calling this workflow from the deploy process.